dev-tools/reconfigurator-cli/tests/output/cmds-target-release-stdout

```diff
@@ -22,6 +22,891 @@ planner config:
+     └─                                                      + artifact: version 0.0.1                                       
+*   nexus             3eeb8d49-eb1a-43f8-bb64-c2338421c2c6   - install dataset           in service    fd00:1122:3344:103::22
+     └─                                                      + artifact: version 0.0.1                                       
+
+
+ COCKROACHDB SETTINGS:
+    state fingerprint:::::::::::::::::   (none) (unchanged)
+    cluster.preserve_downgrade_option:   (do not modify) (unchanged)
+
+ METADATA:
+    internal DNS version:::   1 (unchanged)
+    external DNS version:::   1 (unchanged)
+    target release min gen:   1 (unchanged)
+    nexus gen::::::::::::::   1 (unchanged)
+
+ OXIMETER SETTINGS:
+    generation:   1 (unchanged)
+    read from::   SingleNode (unchanged)
+
+ PENDING MGS UPDATES:
+
+    Pending MGS-managed updates (all baseboards):
+    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    sp_type   slot   part_number   serial_number   artifact_hash                                                      artifact_version   details                                                                                                                                                                                                                                                                                                       
+    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
++   sled      0      model0        serial0         244d553f832cf74043bbcc8a747c8d05384a0f89f7809dcab28c3f707b11f985   0.0.1              Rot(PendingMgsUpdateRotDetails { expected_active_slot: ExpectedActiveRotSlot { slot: A, version: ArtifactVersion("0.0.2") }, expected_inactive_version: NoValidVersion, expected_persistent_boot_preference: A, expected_pending_persistent_boot_preference: None, expected_transient_boot_preference: None })
+
+
+internal DNS:
+  DNS zone: "control-plane.oxide.internal" (unchanged)
+    unchanged names: 51 (records: 65)
+
+external DNS:
+  DNS zone: "oxide.example" (unchanged)
+    unchanged names: 5 (records: 9)
+
+
+
+> # The above run would have triggered an SP update, which we should delete (we want to start
```

Comment:
Not sure how much this matters, but technically the above plan scheduled an RoT update, not SP. Maybe

```suggestion
> # The above blueprint includes a pending MGS update, which we should delete (we want to start
```

(I also wonder if `delete-sp-update` should be `delete-mgs-update`? Happy to file an issue to fix that up separately if you agree.)

Reply:
updated!